lib/scrabble.rb

```diff
@@ -1,11 +1,21 @@
 class Scrabble
+  require 'pry'
 
-  def score(word)
-    1
+  def score("word")
```

Comment:
Parameters of our functions should always be variables.

When we define a function it should never have a string, integer, etc. as the parameter.

```ruby
def score(word)
  # do things to word so that you can return a score.
end
```

When we **call** a function, then we pass it **values** that we want it to use.

```ruby
game = Scrabble.new
game.score("hello")
```

In this case, `"hello"` is a value. We send that when we **call** the function. Not when we define it.